src/lib.rs

```diff
@@ -441,7 +448,7 @@ impl Device {
                         module: desc.vertex_stage.module.id,
                         entry_point: vertex_entry_point.as_ptr(),
                     },
-                    fragment_stage: wgn::PipelineStageDescriptor {
+                    fragment_stage: &wgn::PipelineStageDescriptor {
                         module: desc.fragment_stage.module.id,
```

Comment:
we should make `desc.fragment_stage` to also be an `Option` now

Reply:
Yeah that should have been obvious, sorry!